static/templates/user_info_popover_content.handlebars

```diff
@@ -62,7 +62,10 @@
     {{#if status_text}}
         {{#unless is_me}}<hr />{{/unless}}
         <li class="user_info_status_text">
-            {{status_text}}
+            <span id="status_text">
+                {{status_text}}
+                {{#if is_me}}<a href="#" class="clear_status">(clear)</a>{{/if}}
```

Comment:
we should also put translation tags around the "clear"

Reply:
Woops, sorry about that.